src/presentation/pages/NoteSettings.vue

```diff
@@ -51,7 +60,10 @@ const emit = defineEmits<{
  */
 function onClick(event: MouseEvent) {
   emit('click', event);
-  console.log('TODO: implement noteSettings update');
+  update(props.id, {
+    enabled: noteSettings.value?.enabled ?? false,
+    customHostname: noteSettings.value?.customHostname ?? '',
```

Comment:
when these values can be "undefined" or "null?

Reply:
It is because noteSettings can be `NoteSettings | null`)

Reply:
we can have no response with note settings yet

Reply:
we create them on note creation, so I don't see a case when they can be null